README.md

```diff
@@ -24,17 +24,17 @@
 
 ## Stage 2: Accepted Proposal
 
-**Goal:** Confirm proposal feasibility with Astro Maintainers and TSC.
+**Goal:** Confirm proposal feasibility with Astro Maintainers and [Technical Steering Committee](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#technical-steering-committee-tsc).
 
 **Requirements:** An existing proposal (Stage 1). In addition, a proposal is more likely to be accepted if it is detailed and well thought-out, can demonstrate community interest, has at least one champion volunteer, and has buy-in/interest from Astro maintainer(s).
 
 **Location:** GitHub Issues [(see all accepted proposals).](https://github.com/withastro/roadmap/issues)
 
-**What to Expect:** A proposal reaches this stage (aka "is accepted") during a meeting with Maintainers and TSC, following our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
+**What to Expect:** A proposal reaches this stage (aka "is accepted") during a meeting with Maintainers and [Technical Steering Committee](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#technical-steering-committee-tsc), following our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
```

Comment:
```suggestion
**What to Expect:** A proposal reaches this stage (aka "is accepted") during a meeting with Maintainers and TSC, following our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
```

Reply:
Maybe a footnote with the link could work for all of these? 🤷 if you're not reading this linearly